src/main/java/org/influxdb/InfluxDBFactory.java

```diff
@@ -78,9 +78,30 @@ public static InfluxDB connect(final String url, final OkHttpClient.Builder clie
    */
   public static InfluxDB connect(final String url, final String username, final String password,
       final OkHttpClient.Builder client) {
+    return connect(url, username, password, client, false);
+  }
+
+  /**
+   * Create a connection to a InfluxDB.
+   *
+   * @param url
+   *            the url to connect to.
+   * @param username
+   *            the username which is used to authorize against the influxDB instance.
+   * @param password
+   *            the password for the username which is used to authorize against the influxDB
+   *            instance.
+   * @param client
+   *            the HTTP client to use
+   * @param useMsgPack
+   *            Accept MessagePack format (TRUE) or JSon (FALSE) for response from InfluxDB server
+   * @return a InfluxDB adapter suitable to access a InfluxDB.
+   */
+  public static InfluxDB connect(final String url, final String username, final String password,
```

Comment:
Can we think about a different approach on enabling MsgPack usage? I'm afraid we may end up by having many boolean parameters to activate/deactivate features (for example, we may add support to Protobuf).